src/view/screens/Search/Search.tsx

```diff
@@ -805,6 +805,7 @@ export function SearchScreen(
     } else {
       setSearchText('')
       navigation.setParams({q: ''})
+      textInput.current?.focus()
```

Comment:
Is having this here intentional? Does it do anything in relation to this PR?

Reply:
Sorry this was a rebase mishap on my end, I'll remove this line from the PR as soon as I can 🫡

Reply:
No worries, I can remove!